### DISCUSSION TITLE
is it possible to use vertical split for :RustLsp runnables?

### DISCUSSION THREAD

**Discussion by ssh352** [mrcjkb/rustaceanvim#337] [Q&A] [answered]
Currently the split is a horizontal split below the code.  I `ctrl-w H` to move the split to vertical split on the left,  then I run `:RustLsp runnables`, it uses a horizontal split again. 

Is it possible to configure rustaceanvim to use a vertical split? Thanks!

**Comment by mrcjkb** [comment, marked as answer]
The built-in `'termopen'` runnables executor is very rudimental and can't be configured.
I'm not sure (it came from rust-tools), but it may be possible to do so with the `'toggleterm'` executor. See `:h rustaceanvim.config` and `:h RustaceanToolsOpts`, as well as [the toggleterm docs](https://github.com/akinsho/toggleterm.nvim).
